### DISCUSSION TITLE
你還想在線上考試作弊?

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#525] [General]
清大近期研發出了鷹眼考試防作弊系統，和之前類似功能的系統有了免事先下載、免取得過多的控制權等，這讓隱私權有了比較多的保障。相信等到這類型技術成熟也可以對將來考生之間的公平性有保障。
[https://udn.com/news/story/6928/6196875](https://udn.com/news/story/6928/6196875)

**Comment by HuanYuuuuuu** [comment]
這樣如果我看窗外思考也被判作弊怎麼辦

**Comment by xkllkx** [comment]
萬一要低頭寫字或計算不就被抓了?

**Comment by Yuhsin000** [comment]
據說中國很多公司會在員工的電腦安裝類似的系統，用來記錄員工上班的狀況，甚至還能計算最有可能離職的員工給老闆參考

**Comment by sean1325418** [comment]
感覺電腦鏡頭太爛就不能用眼球辨識，設備會是一個問題

**Comment by PoooYen** [comment]
我們的eeclass就有禁止變換視窗的防作弊系統了哈哈哈

**Comment by jason79461385** [comment]
去年遠距的時候期末考都改成可以開書考了，但難度也都大幅增加⋯ 我倒希望開開視訊鏡頭考平常的難度就好

**Comment by treeann** [comment]
作弊仔真的很討厭 下去吧

**Comment by yi-1111** [comment]
eeclass本來就有防止切換視窗的功能了

**Comment by Jimmy911218** [comment]
這雖然可能會大大減少作弊但是也可能造成其他的小誤會

**Comment by HaoFish123** [comment]
感謝分享，但感覺還蠻嚴格的。

**Comment by alexthebigpp** [comment]
真是讓人恐懼的發明。
